charts/gha-runner-scale-set/values.yaml

```diff
@@ -47,6 +47,20 @@ githubConfigSecret:
 ## name of the runner scale set to create.  Defaults to the helm release name
 # runnerScaleSetName: ""
 
+## A self-signed CA certificate for communication with the GitHub server can be
+## provided using a config map key selector. # If provided, the certificate will
+## also be mounted on runner pods under the # path specified with
+## `runnerMountPath` (defaults to # /usr/local/share/ca-certificates/ if left
+## blank) and NODE_EXTRA_CA_CERTS # environment variable will be set to the path
+## to that certificate to help with # NodeJS-based actions. Example configuration:
```

Comment:
nit: do we want to cleanup these unnecessary `#` ?

Reply:
The idea for these comments in this file is that uncommenting the whole block gives you valid configuration + retains the comment.

Reply:
Even for the second one in the middle of the sentence? 🤔

Reply:
Oooooh my bad 🤦